docs/manual/developer_guide.adoc

```diff
@@ -119,19 +137,19 @@ security content only for *Red Hat Enterprise Linux 7*:
 
 * Take the manual cmake/make approach:
 
-   ```bash
+```bash
    cd content/
    cd build/
    cmake ../
    make -j4 rhel7
-   ```
+```
```

Comment:
You can remove the indent of the commands as well. See how they are rendered below.
![cli_indents](https://user-images.githubusercontent.com/7460169/57914400-d96d0300-788e-11e9-8956-63af5764644f.png)

Reply:
Fixed.